XNAControls/XNAPanel.cs

```diff
@@ -70,6 +83,12 @@ protected override void ParseControlINIAttribute(IniFile iniFile, string key, st
             case "BackgroundTexture":
                 BackgroundTexture = AssetLoader.LoadTexture(value);
                 return;
+            case "BackgroundAnimation":
+                BackgroundAnimation = new Animation(value);
+                BackgroundTexture = BackgroundAnimation.CurrentFrame;
+                Height = BackgroundAnimation.Height;
+                Width  = BackgroundAnimation.Width;
```

Comment:
I don't think height and width should be set here, they aren't set when assigning a regular background texture either.

Reply:
> I don't think height and width should be set here, they aren't set when assigning a regular background texture either.

Thoughts on this solution were in the direction of the modder being able to change the animation resolution after they specifies what animation the client should display. If the animation size was not specified, the default is its natural size.

Reply:
I think if you change the behavior here you also gotta change it on regular background as well for consistency. Or do not change it at all.

Reply:
My suggestion is to keep this code to provide good UX for modders when they do not need to write `Width` and `Height` for animations by default. If they need to change the size of the background animation, they can do it in the `Size`, `Width`, `Height` keys after the `BackgroundAnimation` key.

Since we're talking about XNA UI ini-keys reading, we need to mention exactly the same behaviour as in XNAButton, which I've used as an example: https://github.com/Rampastring/Rampastring.XNAUI/blob/8eb8ecff58ea75b75a9a734b612200389f8c6a0c/XNAControls/XNAButton.cs#L235